ondewo/nlu/agent.proto

```diff
@@ -40,33 +45,46 @@ import "ondewo/nlu/user.proto";
 import "ondewo/nlu/project_role.proto";
 import "google/protobuf/timestamp.proto";
 
-// Agents are best described as Natural Language Understanding (NLU) modules
-// that transform user requests into actionable data. You can include agents
-// in your app, product, or service to determine user intent and respond to the
-// user in a natural way.
```

Comment:
@ficinator 
Any reason why it should be in a single line rather than several?

Reply:
Filip: "Having multi-line was creating a new paragraph for each line in Markdown. However in HTML should be fine. We should experiment a bit with it."